src/apis/us/UserCommHandler.java

```diff
@@ -0,0 +1,43 @@
+package apis.us;
+
+public interface UserCommHandler {
+
+    String outPutLocation=null;
+    String inputType=null;
+    //file or command line or scanner.
+    
+    
+    
+    public void getUserInput();
+    //MAKE SURE setOutPut type is set??
+    //if outPutLocation==null, set it to system.out.println
+    //if inputType (Delimiters) is not set, call it to make it up!!
+    //available options
+    //scanner
+    //command line
+    //???
+    
+    
+    
+    public void setDelimiters();
+    //prompt the user for delimiters
+    //SETS INPUT TYPE
+    //Use overflowing
+    
+    public String setOutputType(String s);
+    //output to command line
+    //Or a file
+    //IF ITS A FILE, RETURN FILENAME??
+    
+    
+    public void outPut(String stringParam);
```

Comment:
unclear what this is doing exactly. Does it print the string?

Reply:
I think you may have missed the comments underneath this method, it prints the result in the previously specified space. Either a file, CSV, or in the console